src/main/java/codegym/tequila/fisioapp/model/PhysicalExploration.java

```diff
@@ -0,0 +1,49 @@
+package codegym.tequila.fisioapp.model;
+
+import jakarta.persistence.*;
+
+@Embeddable
+public class PhysicalExploration {
+
+    @Column(columnDefinition = "DECIMAL(5, 2)")
```

Comment:
As we are creating the DB structure via a DDL, I recommend to omit the column definition here, the logic behind this recomendation is: If the field needs to be updated let's say to 6 integer positions, we can just change the database without touching Java code